File: #171_Excel_Sheet_Column_Number/Solution.java
public class Solution {
    public int titleToNumber(String s) {
        int result = 0;
        for (int i = 0; i < s.length(); i++) {
            result = result * 26 + (s.charAt(i) - 'A' + 1);
        }
        return result;
    }
    public static void main(String[] args) {
    	Solution sol = new Solution();
    	System.out.println(sol.titleToNumber("AA"));
    }
}

File: #171_Excel_Sheet_Column_Number/Solution.py
class Solution:
    # @param {string} s
    # @return {integer}
    def titleToNumber(self, s):
        ans = 0
        for c in s:
        	ans = ans * 26 + ord(c) - ord('A') + 1
        return ans

sol = Solution()
print sol.titleToNumber("AA")